pkg/storaged/stratis-details.jsx

```diff
@@ -207,6 +217,140 @@ export const StratisPoolDetails = ({ client, pool }) => {
         });
     }
 
+    function add_passphrase() {
+        dialog_open({
+            Title: _("Add passphrase"),
+            Fields: [
+                PassInput("passphrase", _("Passphrase"),
+                          { validate: val => !val.length && _("Passphrase cannot be empty") }),
+                PassInput("passphrase2", _("Confirm"),
+                          { validate: (val, vals) => vals.passphrase.length && vals.passphrase != val && _("Passphrases do not match") })
+            ],
+            Action: {
+                Title: _("Save"),
+                action: vals => {
+                    return get_unused_keydesc(client, pool.Name)
+                            .then(keydesc => {
+                                return with_stored_passphrase(client, keydesc, vals.passphrase,
+                                                              () => pool.BindKeyring(keydesc))
+                                        .then(std_reply);
+                            });
+                }
+            }
+        });
+    }
+
+    function change_passphrase() {
+        with_keydesc(client, pool, (keydesc, keydesc_set) => {
+            dialog_open({
+                Title: _("Change passphrase"),
+                Fields: [
+                    PassInput("old_passphrase", _("Old passphrase"),
+                              {
+                                  visible: vals => !keydesc_set,
+                                  validate: val => !val.length && _("Passphrase cannot be empty")
+                              }),
+                    PassInput("new_passphrase", _("New passphrase"),
+                              { validate: val => !val.length && _("Passphrase cannot be empty") }),
+                    PassInput("new_passphrase2", _("Confirm"),
+                              { validate: (val, vals) => vals.new_passphrase.length && vals.new_passphrase != val && _("Passphrases do not match") })
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18818-20230707-100043-8d39ae7d-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)